apps/explorer-e2e/src/helpers/getExploredTip.ts

```diff
@@ -0,0 +1,9 @@
+import Axios from 'axios'
+import { clusterd } from '@siafoundation/clusterd'
+
+export async function getExploredTip() {
+  const response = await Axios.get(
+    `http://localhost:${clusterd.managementPort}/nodes/proxy/explored/api/explorer/tip`
```

Comment:
This should be the same as using the existing sdk `cluster.daemons.explored.api.explorerTip()`